lib/acme/client.rb

```diff
@@ -127,9 +128,28 @@ def finalize(url:, csr:)
     Acme::Client::Resources::Order.new(self, **arguments)
   end
 
-  def certificate(url:)
+  def certificate(url:, preferred_chain: nil)
     response = download(url, format: :pem)
-    response.body
+    pem = response.body
+    identifier_class = Acme::Client::ChainIdentifier
```

Comment:
Why the local over just using `ChainIdentifier` directly? We're in the `Acme::Client` namespace, so no need to prefix it. The local made me wonder if this is for some reason is configurable.

Reply:
You are right, no need to fully qualify here.